server/chat-plugins/trivia.ts

```diff
@@ -130,17 +130,16 @@ function isTriviaRoom(room: Room) {
 	return false;
 }
 
-function getTriviaGame(context: CommandContext) {
-	const room = context.room;
-	if (!room) return false;
+function getTriviaGame(room: Room) {
+	if (!room) {
```

Comment:
I'd probably remove the braces on these first two `if`s now that theyre fairly short

Reply:
I'd rather have them all match.